3rd-party-tools/open-fortran-parser-sdf/trans/Makefile

```diff
@@ -0,0 +1,74 @@
+# Usage:
+#
+# If the OFP_HOME environment variable contains the path to the Open Fortran Parser,
+# then this Makefile transforms the coarray Fortran (CAF) source defined in SOURCE 
+# environment variable into the OpenCoarrays CAF (OCAF) dialect printed to the 
+# standard output.  Most commonly, one sets SOURCE on the command line that invokes
+# "make":
+#
+#    make SOURCE=foo.f90 check > ocaf-foo.f90
+#    make clean
+#
+# Alternatively, one can set OFP_HOME also in the "make" invocation: 
+#
+#    make SOURCE=foo.f90 OFP_HOME=/opt/ofp-sdf check >  ocaf-foo.f90
+#    make OFP_HOME=/opt/ofp-sdf clean
+#
+
+ifeq ($(OFP_HOME),)
+  $(error Usage: make SOURCE=foo.f90 OFP_HOME=/path/to/ofp-sdf check >  ocaf-foo.f90)
+endif
+
+include $(OFP_HOME)/make.inc
```

Comment:
I agree it would be very desirable to find a better approach than the current Makefile.  I expect to talk with Craig this afternoon to at least get the current set-up working.  Any time you can put into improving the build system upstream or using git submodules or CMake or anything better than the current set-up would be very much appreciated.  My next steps with Craig are

1. Determining what files from OFP need to be copied into the ultimate installation folder in order for OpenCoarrays to use OFP without keeping the entire OFP source tree around.
2. Figuring out how to invoke OFP from the new caf2ocaf.sh script that gets installed by [line 78 in src/mpi/CMakeLists.txt](https://github.com/sourceryinstitute/opencoarrays/blob/add-ofp/src/mpi/CMakeLists.txt#L78) in my latest commit.